### PR TITLE
Feature/dimension values

### PR DESCRIPTION
**Pull request by diericd**

### CODE REVIEW

File: pkg/util/periodic_property_values.go [hunk 1]
```diff
@@ -1,66 +1,180 @@
 package util
 
-import "slices"
+import (
+	"encoding/json"
+
+	"sort"
+
+	"golang.org/x/exp/maps"
```
Comment (vdpoora) [inline pkg/util/periodic_property_values.go:8]: pls use stdlib "maps"

File: src/QueryEditor/util.ts [hunk 2]
```diff
@@ -366,3 +367,10 @@ export function semverCompare(a: string, b: string): number {
 
   return 0
 }
+
+export function isSupportedPrototypeType(type: PropertyType, version: string): boolean {
+  if (type === PropertyType.PeriodicWithDimension && semverCompare(version, 'v7.2.0') < 0) {
```
Comment (vdpoora) [inline src/QueryEditor/util.ts:372]: also allow version == "debug"

File: src/CustomVariableEditor/EventTypePropertyFilter.tsx
Comment (vdpoora) [inline src/CustomVariableEditor/EventTypePropertyFilter.tsx:1]: image

This list should show only the periodic_with_dimension event props right?

**Comment by vdpoora** [comment]
image


tooltip should only have simple properties

**Comment by vdpoora** [comment]
Would be cool that when a dashboard visualisation is made, by default the trendspanel is chosen as visualisation if the user selects the PeriodicWithDimension querytype

**Comment by diericd** [comment]
> Would be cool that when a dashboard visualisation is made, by default the trendspanel is chosen as visualisation if the user selects the PeriodicWithDimension querytype

can you create a story for this.

**Review by vdpoora — APPROVED** [review]

**Pull request merged**